build/dnsmasq/local.mog

```diff
@@ -0,0 +1,20 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+# Copyright 2022 OmniOS Community Edition (OmniOSce) Association.
+
+dir group=$(GROUP) mode=0770 owner=$(USER) path=var/$(BASEDIR)
+
+group groupname=$(GROUP) gid=99
+user ftpuser=false username=$(USER) uid=99 group=$(GROUP) \
```

Comment:
UIDs are tracked in the `doc/idlist.md` file in this repository, covering the users that come from illumos-omnios, omnios-build as well as extra.
99 is free, but I'd suggest using 56 since that groups it with the other DNS services
(and please add rows to that file).